Header header
geometry_msgs/Vector3 thrust
geometry_msgs/Vector3 torque
float64[] rotor_speeds